website/docs/guides/data_source_filters.html.markdown

```diff
@@ -44,13 +44,16 @@ saved into a catalog. See Example 7 below.
 When a filter contains multiple clauses, you achieve the overall match only if all the clauses match. For example:
 
 ```hcl
+# TF-UPGRADE-TODO: Block type was not recognized, so this block and its contents were not automatically upgraded.
```

Comment:
This was injected by the tool here and I guess it doesn't make sense as it is just an excerpt, not a complete HCL.

Reply:
I will use the opportunity for one more tiny thing. This is a very good user facing cleanup and it deserves a changelog note :)

Could you add a `705-improvements.md` file to `.changes/v3.4.0` and put inside a note. Feel free to grab inspiration from any other notes files there. 

Thank you!

Reply:
Ping!

Reply:
Thanks for the contributoin - I am going to merge it and add a changelog note in a separate pr.